src/index.ts

```diff
@@ -292,9 +307,7 @@ function isAllowedToExecute(binding: ComboBinding, e: KeyboardEvent): boolean {
 	) {
 		return false
 	} else if (typeof binding.global === 'function') {
-		if (!binding.global(e, stringifyCombo(binding.combo))) {
```

Comment:
Since `binding.global` is an external function that Sorensen can't rely on having any particular return type (someone might be using plain JavaScript and no type checking), I think I would like to keep relying on this function returning either `true` or `false` and not passing through the return value of `binding.global`, since there's a risk this may lead to undefined behavior elsewhere.

Reply:
That makes sense. I have commit a change, where it is doubled negated instead.